doc/Tribler REST API.md

```diff
@@ -64,6 +70,18 @@ Returns all discovered channels in Tribler.
 }
 ```
 
+## `POST /channel/:channelcid/subscribe`
```

Comment:
Hmm it indicates that the channel cid should be substituted in there. I replaced the `:` with `{channelid}`.